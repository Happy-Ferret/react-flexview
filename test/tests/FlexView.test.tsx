import * as React from 'react';
import * as renderer from 'react-test-renderer';
import FlexView from '../../src';

describe('FlexView', () => {

  it('renders correctly', () => {
    const component = renderer.create(
      <FlexView
        vAlignContent='center'
        hAlignContent='right'
        grow
        shrink
        wrap
      >
        CONTENT
      </FlexView>
    );
    expect(component).toMatchSnapshot();
  });

});
