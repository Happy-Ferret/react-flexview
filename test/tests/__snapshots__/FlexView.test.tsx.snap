// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexView renders correctly 1`] = `
<div
  className="react-flex-view align-content-center justify-content-end flex-wrap"
  style={
    Object {
      "MozBoxFlex": "1 1 auto",
      "WebkitBoxFlex": "1 1 auto",
      "WebkitFlex": "1 1 auto",
      "flex": "1 1 auto",
      "msFlex": "1 1 auto",
    }
  }
>
  CONTENT
</div>
`;
